resources/views/home.blade.php

```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ベトナム料理</title>
+      <!-- font google -->
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700&display=swap" rel="stylesheet">
+  <!-- font-awesome -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css">
+  <!-- main css -->
+    <link rel="stylesheet" href="css/home.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboro:300,400,500,700">
+</head>
+<body>
+    
+    <h5>Choose laguage</h5>
+    <ul class="list-group list-group-horizontal">
+        <a class="list-group-item" href="/en">English</a>
+        <a class="list-group-item" href="/vn">Vietnamese</a>
+        <a class="list-group-item" href="/jp">Japanese</a>
```

Comment:
Đặt tên cho route và dùng hàm route() thay vì URL.
```suggestion
        <a class="list-group-item" href="{{ route('locale', 'en') }}">{{ __('lang.english') }}</a>
        <a class="list-group-item" href="{{ route('locale', 'vi') }}">{{ __('lang.vietnamese') }}</a>
        <a class="list-group-item" href="{{ route('locale', 'jp') }}">{{ __('lang.japanese') }}</a>
```